pylint/checkers/variables.py

```diff
@@ -1483,7 +1465,6 @@ def _check_consumer(
         consumer_level: int,
         base_scope_type: Any,
     ) -> VariableVisitConsumerActionAndOptionalNodesType:
```

Comment:
Why did you remove the docstring here?

Reply:
Oops, that was an oversight. Thanks for catching.